scopesim_templates/metis/laser.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""WCU laser spectrum for METIS."""
+
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+
+from spextra import Spextrum
+
+from ..rc import Source, im_plane_utils
+
+
+def laser_spectrum(img_size=(12, 12),
+                   waves=np.arange(3, 6, .001)*u.um,
+                   centers=[3.39, 4.73, 5.263]*u.um,
```

Comment:
From the template manual:

> The three L and M band laser sources in the WCU are: (1) a fixed laser at 3.39 micron, (2) a tuneable laser at 4.68 to 4.78 micron and (3) a fixed laser at 5.26 micron.
> 
> The laser can be tuned quickly several times during a long exposure with the long-slit spectrograph so that a single exposure with multiple lines is created.

So perhaps we can update the default to have more than 3 lines? Say 10 between 4.68 and 4.78 micron? I don't know how many would be realistic.

Reply:
I took all the numbers from AstarVienna/METIS_Simulations#11. But I would be trivial to include multiple "variants" for the tunable laser.